### DISCUSSION TITLE
Does AdguardHome support `random_padding`?

### DISCUSSION THREAD

**Discussion by c2xusnpq6** [AdguardTeam/AdGuardHome#3370] [Q&A]
https://developers.google.com/speed/public-dns/docs/doh/json

> random_padding
string, ignored
>
>The value of this parameter is ignored. Example: XmkMw~o_mgP2pf.gpw-Oi5dK.
>
>API clients concerned about possible side-channel privacy attacks using the packet sizes of HTTPS GET requests can use this to make all requests exactly the same size by padding requests with random data. To prevent misinterpretation of the URL, restrict the padding characters to the unreserved URL characters: upper- and lower-case letters, digits, hyphen, period, underscore and tilde.

![image](https://user-images.githubusercontent.com/75058881/126675569-8c80353b-d60e-42d3-b5ab-5e32cfaa5fcf.png)

**Comment by c2xusnpq6** [comment]
one more... Does Adguard DNS support `random_padding`?

**Comment by ainar-g** [comment]
As far as I know, no, and it doesn't support DoH JSON API at all.
